interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a program meant to store multiple versions of code so that you're able to track different versions of that code for bug solving and testing purposes. Git stands for Global Information Tracker, and it's the platform that Github runs on. So, Github is an interface that allows us to use the program called Git.
```

Comment:
Incredible work in this section!  You answers are extremely thorough and well worded.  Great job!!